react/w-books/src/app/components/Button/index.tsx

```diff
@@ -1,27 +1,30 @@
 import clsx from 'clsx';
 import React, { ReactNode } from 'react';
 
-import styles from './styles.module.scss';
-
 interface ButtonProps {
   children: ReactNode;
   className?: string;
   isFilled?: boolean;
   isSubmit?: boolean;
+  isWidthAuto?: boolean;
+  handleClick?: () => void;
 }
 
-export default function Button({ children, isSubmit, className, isFilled }: ButtonProps) {
+export default function Button({ children, isSubmit, className, isFilled, isWidthAuto, handleClick }: ButtonProps) {
 
   return (
     <button
       type={isSubmit ? 'submit' : 'button'}
+      onClick={handleClick}
       className={clsx(
-        styles.button,
+        'button',
```

Comment:
You may have several types of btn in a application. I reccomend calling this one `primary-btn`